tests/Integration/goldens/iam/samples/V1/IAMPolicyClient/get_iam_policy.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+
+// [START iam-meta-api_v1_generated_IAMPolicy_GetIamPolicy_sync]
```

Comment:
This should be iam only, see b/247779318

```suggestion
// [START iam_v1_generated_IAMPolicy_GetIamPolicy_sync]
```

Also take care to remove the region from a regionalized host if any, [here](https://github.com/googleapis/gapic-generator-csharp/blob/7737012d6f17708c77326059cb9cb0a282e33e59/Google.Api.Generator/Generation/ServiceDetails.cs#L69-L72) the .NET code.